### PR TITLE
feat(Cuevana 3): fix url

### LINKED ISSUE #6351: Cuevana3 | https://ww3.cuevana.pro/
### Steps to Reproduce

https://ww3.cuevana.pro/
https://premid.app/store/presences/Cuevana%203

### Expected Behavior

that the presence of cuevana3 will function normally as the other premid 

### Actual Behavior

cuevana3 premid site is not working 

### Page URL

https://ww3.cuevana.pro/

### Screenshots

Screenshot 2022-06-21 203544

### Operating System

windows 10 home

### Browser and Version

brave, google 

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by realtofuine**
## Description 
change url of cuevana3.me to ww1.cuevana3.me.

I'm not entirely sure if this is an issue others are experiencing, but at least for me it only works with the new changed url.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

image

image

</details>

### CODE REVIEW

**Comment by DiegoYisus** [comment]
It still does not work:(

**Comment by realtofuine** [comment]
> It still does not work:(

did you use the modified version?

i dont believe that the one downloaded from premid store will be changed until this pull request is approved

**Comment by DiegoYisus** [comment]
Yes, use the modified link that you sent but still does not work even with the link of the official premid of the cuevana

**Comment by ImRodry** [comment]
@DiegoYisus did you clone the repository and manually load this version of the presence? The author didn’t send any special links I’m not sure what you mean

**Comment by DiegoYisus** [comment]
yes i used that version

**Comment by DiegoYisus** [comment]
what I mean is that I use the modified link that first one commented on the post but I used that one and it didn't work

**Comment by ImRodry** [comment]
Like I said, that kink isn’t special at all, it links to the latest published version of this presence which doesn’t have the fix. You’d have to clone the repository and checkout this branch in order to use it

File: websites/C/Cuevana 3/presence.ts [hunk 1]
```diff
@@ -30,40 +30,103 @@ presence.on(
...
+				presenceData.buttons = [
+					{ label: "Ver Episodio", url: window.location.href },
+				];
+				presenceData.largeImageKey = cover;
+			}
+
+			if (iFrameVideo) {
+				(presenceData.smallImageKey = videoPaused ? "pause" : "play"),
+					(presenceData.smallImageText = videoPaused
+						? (await strings).pause
```
Comment (EncryptedDev) [inline websites/C/Cuevana 3/presence.ts:73]: Use the Awaited type for strings and remove the await here as shown below
https://github.com/PreMiD/Presences/blob/5e5f99668ba4800524e66c2a3fc9fd38755025c6/websites/G/Giggl/presence.ts#L19

**Review by EncryptedDev — APPROVED** [review]

File: websites/C/Cuevana 3/presence.ts [hunk 2]
```diff
@@ -30,47 +33,112 @@ presence.on(
...
 
 		if (iFrameVideo) {
 			(presenceData.smallImageKey = videoPaused ? "pause" : "play"),
 				(presenceData.smallImageText = videoPaused
-					? (await strings).pause
-					: (await strings).play),
+					? strings.pause
+					: strings.play),
 				(presenceData.startTimestamp = startTimestamp),
 				(presenceData.endTimestamp = endTimestamp);
```
Comment (EncryptedDev) [inline websites/C/Cuevana 3/presence.ts:143]: The extra brackets are not needed

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/C/Cuevana 3/presence.ts (hunk 2 above)
Comment (EncryptedDev) [reply websites/C/Cuevana 3/presence.ts:143]: 🤔

Comment (realtofuine) [reply websites/C/Cuevana 3/presence.ts:143]: how am i this bad

Comment (realtofuine) [reply websites/C/Cuevana 3/presence.ts:143]: oh i think it messed up when i clicked add commit on github

Comment (realtofuine) [reply websites/C/Cuevana 3/presence.ts]: OH @EncryptedDev yarn format likes it like that

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Cheers for the PR 🎉

**Pull request merged**